### PR TITLE
GH issue #385: added para on logoutURL

### PR DESCRIPTION
**Pull request by bfallonf**
@liggitt This PR is adding the info from #385 to the docs.

Is there anything else you'd like included in this PR? Anything I'd be missing?

Thanks.

### CODE REVIEW

File: admin_guide/web_console_customization.adoc [hunk 1]
```diff
@@ -224,3 +224,25 @@ oauthConfig:
...
+
+You can change the location console users will be sent to upon logging out of
+the console by modifying the `*logoutURL*` parameter in the
+*_/etc/openshift/master/master-config.yaml_* file:
+
+====
+----
+...
+assetConfig:
+  logoutURL: "www.example.com"
```
Comment (liggitt) [inline admin_guide/web_console_customization.adoc]: prefix with https://

File: admin_guide/web_console_customization.adoc [hunk 2]
```diff
@@ -224,3 +224,25 @@ oauthConfig:
...
+...
+----
+====
+
+This can be useful when authenticating with
+link:configuring_authentication.html#RequestHeaderIdentityProvider[Request
+Header] and oauth or
```
Comment (liggitt) [inline admin_guide/web_console_customization.adoc]: OAuth

File: admin_guide/web_console_customization.adoc [hunk 3]
```diff
@@ -224,3 +224,25 @@ oauthConfig:
...
+link:configuring_authentication.html#OpenID[openid] identity
+providers, as sending console users to an external URL upon logout destroys
```
Comment (liggitt) [inline admin_guide/web_console_customization.adoc]: providers which require visiting an external URL to destroy their SSO sessions.

File: admin_guide/web_console_customization.adoc [hunk 4]
```diff
@@ -224,3 +224,25 @@ oauthConfig:
...
+----
+...
+assetConfig:
+  logoutURL: "www.example.com"
+...
+----
+====
+
+This can be useful when authenticating with
+link:configuring_authentication.html#RequestHeaderIdentityProvider[Request
```
Comment (adellape) [inline admin_guide/web_console_customization.adoc]: Give the full path per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references

**Comment by bfallonf** [comment]
@liggitt @adellape Thanks for the comments. Addressed. If nothing else, this should be OK to merge.

**Pull request merged**